lib/stdlib/src/log_util.rs

```diff
@@ -116,15 +116,12 @@ pub(crate) static REGEX_NGINX_COMBINED_LOG: Lazy<Regex> = Lazy::new(|| {
         \-\s+                                   # Always a dash
         (-|(?P<user>\S+))\s+                    # Match `-` or any non space character
         \[(?P<timestamp>.+)\]\s+                # Match date between brackets
-        "(?P<request>
+        "(?P<request>[^"]*)"\s+                 # Match any non double-quote character
         (?P<method>\w+)\s+                      # Match at least a word
-        (?P<path>\S+)\s+                        # Match any non space character
```

Comment:
The type definitions of the `parse_nginx_log` need to change to remove `method`, `path`, and `version`. The other similar regexes should also be changed to be consistent, such as `REGEX_INGRESS_NGINX_UPSTREAMINFO_LOG`.

Reply:
agreed